internal/opentelemetry/handlers/k8s.go

```diff
@@ -0,0 +1,83 @@
+package handlers
```

Comment:
These changes were part of the previous approach where we relied on just checking for the objects that were specified in the configuration. I'll maybe move this to a separate PR.